packages/eslint-plugin-lit-a11y/lib/rules/aria-attr-types.js

```diff
@@ -122,7 +137,7 @@ module.exports = {
                 ariaAttributes.forEach(attr => {
                   const normalizedName = attr.toLowerCase();
                   // Not a valid aria-* state or property.
-                  if (aria.get(normalizedName) === undefined) {
+                  if (!isAriaPropertyName(normalizedName)) {
```

Comment:
shouldn't we report an error here? or just leave it to `aria-attr`

Reply:
this is covered by `aria-attr` rule, so yeah

Reply:
so this is resolved right?